sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+tasks. Random forests, which cannot take advantage of the related training
+tasks, trained in the same setting achieve near random performance.  Despite the
+success on Tox21, performance on MUV datasets, which contains assays designed to
+be challenging for chemical informatics algorithms, is considerably worse.  The
+authors also demonstrate the limitations of transfer learning as embeddings
+learned from the Tox21 assays have little utility for a drug adverse reaction
+dataset.
+
+In the long term, these novel, learned chemical feature representations may
+prove to be essential for accurately predicting why some compounds with similar
+structures yield similar target effects and others produce drastically different
+results.  Currently, these methods are enticing but do not necessarily
+outperform classic approaches by a large margin.  The neural fingerprints
+[@tag:Duvenaud2015_graph_conv] were narrowly beaten by regression using
+traditional circular fingerprints on a drug efficacy prediction task (but were
+superior for predicting solubility or photovoltaic efficiency).  Graph
+convolutions [@tag:Kearnes2016_graph_conv] performed comparably to a multitask
+network using standard fingerprints and slightly better than the neural
+fingerprints [@tag:Duvenaud2015_graph_conv] on the drug efficacy task but were
+slightly worse than the influence-relevance voter method on an HIV dataset.
+[@tag:Swamidass2009_irv].  `TODO: there are also problems with some papers using
+ROC primarily for benchmarking, which skews results and makes it hard to assess
+absolute performance, but I already alluded to that above and may leave further
+discussion to the Discussion section`
+
+We remain optimistic for the potential of deep learning and specifically
+representation learning in this domain and propose that rigorous benchmarking on
+broad and diverse prediction tasks will be as important as novel neural network
+architectures to advance the state of the art _and convincingly demonstrate
+superiority over traditional cheminformatics techniques_. Fortunately, there has
+recently been significant progress in this direction. The DeepChem software
+[@tag:AltaeTran2016_one_shot @url:https://github.com/deepchem/deepchem] and
+MoleculeNet benchmarking suite [@tag:Wu2017_molecule_net] built upon it contain
+chemical bioactivity and toxicity prediction datasets, multiple compound
+featurization approaches including graph convolutions, and various machine
+learning algorithms ranging from standard baselines like logistic regression and
+random forests to recent neural network architectures.  Independent research
+groups have already contributed additional datasets and prediction algorithms,
+and adoption of common benchmarking evaluation metrics, datasets, and baseline
+algorithms has the potential to establish the practical utility of deep learning
+in chemical bioactivity prediction and lower the barrier to entry for machine
```

Comment:
Added `to DeepChem` to clarify it is indeed the above resource.  It's not quite a fully open community resource in that I think the main repository owners are mostly (entirely?) from one lab, but I did want to highlight community engagement and the eagerness to accept contributions from others.